src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -70,7 +75,7 @@ export const UserOrcid = ({ user }: UserOrcidProps) => {
     if (orcidAccessToken) {
       addOrcid(orcidAccessToken);
     }
-  }, [t, dispatch, history, userCristinId, refetchCristinPerson]);
+  }, [t, dispatch, history, userCristinId, cristinPersonQuery]);
```

Comment:
teste med å fjerne og legge til orcid, og dette virker fortsatt uten at det blir noen evig loop eller noe sånn?

Reply:
Var smertefritt for meg å fjerne og legge til orcid på nytt ;)